installers/installer.iss

```diff
@@ -26,7 +26,7 @@ UsedUserAreasWarning=no
 LicenseFile=license.txt
 ArchitecturesInstallIn64BitMode=x64
 OutputBaseFilename=setupSasView
-SetupIconFile=dist\sasview\_internal\images\ball.ico
+SetupIconFile=dist\sasview\images\ball.ico
```

Comment:
These fixes for `_internal` directory look like they belong to a separate issue/PR?